draft-packages/modal/KaizenDraft/Modal/Presets/InputEditModal.tsx

```diff
@@ -40,6 +39,7 @@ const InputEditModal = ({
   submitWorking,
   automationId,
   children,
+  unpadded = false,
```

Comment:
If the default is to be padded, I'd probably prefer this is to be `padded` to save the double-negative here. Unless you're working off of alternative advice already though!

Reply:
@lloydstubber  Yep makes sense! I think my thought was when using each of the components the consumer will now have to add `padded` as a prop to ensure its padded. whereas they would only need to add the prop if they specifically want it unpadded. 

I wonder... Do we need an unpadded / padded prop? This was something that was exisiting already in the components which I continued with these updates.

Reply:
Ahh okay that does make sense then if it's optional and hardly used. I was going more on the readability but what you have makes sense/seems to be what is dotted around the Modal component already. Nice one 👍

Reply:
I recently had to create a prop like this and @ActuallyACat and I opted for the name `noPadding`. Soz can we please change this for consistency?